app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -3,10 +3,6 @@ module ManageIQ::Providers::Lenovo
   class PhysicalInfraManager::RefreshParser < EmsRefresh::Parsers::Infra
     include ManageIQ::Providers::Lenovo::RefreshHelperMethods
 
-    def self.miq_template_type
-      parent::Parser::ParserDictionaryConstants::MIQ_TYPES["template"]
```

Comment:
Can you delete this from ParserDictionaryConstants also?

Reply:
This is deleted on the next PR (https://github.com/ManageIQ/manageiq-providers-lenovo/pull/183) that manages the dictionary constants